webclient/static/js/mapa_doc.js

```diff
@@ -122,7 +122,7 @@ switchMap = function (overview = false) {
             console.log("Change: " + northWest + "  " + southEast + " " + zoom);
             boundsLock = bounds;
             let xhr_3d_all = new XMLHttpRequest();
-            xhr_3d_all.open('POST', '/dokument/akce-get-3d');
+            xhr_3d_all.open('POST', '/dokument/mapa-3d');
```

Comment:
v kontextu https://github.com/ARUP-CAS/aiscr-webamcr/issues/301#issuecomment-1802682882 - zde myslím stačí změnit na `dokument/model/mapa-3d` a je to